docs/source/man.rst

```diff
@@ -204,7 +204,7 @@ Sources
 
 Use the ``qpc source`` command to create and manage sources.
 
-A source defines a collection of network information, including IP addresses or host names, or systems management solution information, in addition to SSH ports and SSH credentials. The SSH credentials are provided through reference to one or more credentials. A scan can reference a source so that the act of running the scan is repeatable, without a requirement to reenter network information for each scan attempt.
+A source defines a single entity or multiple entities to be inspected. A source can be a single physical machine, virtual machine, or container, or it can be a collection of network information, including IP addresses or host names, or systems management solution information such as vCenter Server or Satellite information. The source also contains information about SSH ports and SSH credentials that are needed to access the systems to be inspected. The SSH credentials are provided through reference to one or more of the Quipucords credentials that you configure. A scan  references a source so that the act of running the scan is repeatable, without a requirement to reenter the information for each scan attempt.
```

Comment:
It looks like there is an extra space in the last sentence after `A scan`. It is up to you if you want to fix this or not, as I don't think it will show up once it's rendered.

Reply:
FYI, I changed the text of some of these old definitions of major objects because they had been added to over time and needed a slight overhaul--per discussions with Elijah in some of these other comments. Thanks for mentioning the extra space, though, those are frequently hard to spot.